src/content/docs/de/core-concepts/sharing-state.mdx

```diff
@@ -0,0 +1,694 @@
+---
+title: Status zwischen Astro-Inseln teilen
+description: Lerne, wie du den Status von Komponenten - und Frameworks! - mit Nano Stores teilst.
```

Comment:
Würde vielleicht “Zustand” besser passen als “Status”?
```suggestion
title: Zustand zwischen Astro-Inseln teilen
description: Lerne, wie du den Zustand von Komponenten - und Frameworks! - mit Nano Stores teilst.
```
(Die [Übersetzung von den React-Dokumentation](https://de.reactjs.org/docs/state-and-lifecycle.html) benutzt komischerweise “State” aber ich glaube, dass eine Übersetzung hier besser ist.)

Reply:
Yes, that's a better term, will change it.